docs/how-to-guides/advanced-user-guides/monitoring-alerting-guides/enable-monitoring.md

```diff
@@ -77,3 +77,79 @@ key.pfx=`base64-content`
 ```
 
 Then **Cert File Path** would be set to `/etc/alertmanager/secrets/cert.pem`.
+
+## Rancher Performance Dashboard
+
+When monitoring is installed, you are given basic health metrics about the local Rancher server, such as CPU and memory data. To get advanced metrics for your local Rancher server, you must enable the Rancher Performance Dashboard for Grafana.
+
+This dashboard provides access to the following advanced metrics:
+
+- Handler Average Execution Times Over Last 5 Minutes
+- Rancher API Average Request Times Over Last 5 Minutes
+- Subscribe Average Request Times Over Last 5 Minutes
+- Lasso Controller Work Queue Depth
+- Number of Rancher Requests
+- Number of Failed Rancher API Requests
+- K8s Proxy Store Average Request Times Over Last 5 Minutes
+- K8s Proxy Client Average Request Times Over Last 5 Minutes
+- Cached Objects by GroupVersionKind
+- Lasso Handler Executions
+- Handler Executions over last 2 minutes
+- Total Handler Executions with Error
+- Data Transmitted by Remote Dialer Sessions
+- Errors for Remote Dialer Sessions
+- Remote Dialer Connections Removed
+- Remote Dialer Connections Added by Client
```

Comment:
Are all of these, capitalization included, directly copied from the UI? Some of the capitalization is inconsistent (line 97 has "over" in lowercase while other metrics use "Over")

Reply:
Double checked dashboard names and updated. That one was a typo.